### ISSUE TITLE
nexus-prisma doesn't support id block

### ISSUE THREAD

**Issue by codingfunwoody** [graphql-nexus/nexus-plugin-prisma#628]
```
model Staff {
  email       String
  upn         String
  displayName String
  @@id([email, upn])
}
```

Error: Your Prisma Model Staff does not have an @id field. It's required for nexus-prisma to work.

**Comment by gabrielbusarello** [comment]
Something new about that?

**Comment by Weakky** [comment]
Thanks for reporting. Closed by #686. Will be in the next release.
